### PR TITLE
aws file upload

### PR DESCRIPTION
**Pull request by ghost**
@abhilashsajeev @sam-qburst @AmalQB @madhukarmmallia

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -11,8 +11,8 @@ var users = require('./routes/users');
 var app = express();
 
 // view engine setup
-app.set('views', path.join(__dirname, 'views'));
-app.set('view engine', 'jade');
+// app.set('views', path.join(__dirname, 'views'));
```
Comment (abhilashsajeev) [inline app.js]: @nithinp Remove commented codes

File: app.js [hunk 2]
```diff
@@ -11,8 +11,8 @@ var users = require('./routes/users');
...
+// app.set('view engine', 'jade');
 
 // uncomment after placing your favicon in /public
 //app.use(favicon(path.join(__dirname, 'public', 'favicon.ico')));
```
Comment (abhilashsajeev) [inline app.js]: @nithinp remove commented codes.

File: config/constants.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+var k = {
+  AWS : {
+    BUCKET_NAME : 'my_bucket',
+    ACCESS_KEY_ID: 'ACCESS_KEY_ID',
+    SECRET_ACCESS_KEY: 'SECRET_ACCESS_KEY',
+    ENDPOINT: 'http://localhost:4567'
+  }
+};
+
+module.exports = k;
```
Comment (abhilashsajeev) [inline config/constants.js]: you can use module.export directly

``` javascript
module.exports = {
AWS: {}
}

```

File: routes/index.js [hunk 4]
```diff
@@ -2,12 +2,49 @@
...
 /* GET home page. */
 router.get('/', function (req, res, next) {
-  res.render('index', {
-    title: 'Express'
+  var filePath = path.resolve(__dirname + '/../public/home.html');
+  fs.createReadStream(filePath).pipe(res);
+});
+
+router.post('/', upload.single('file'), function (req, res, next) {
+  //console.log(req.file);
```
Comment (abhilashsajeev) [inline routes/index.js]: @nithinp remove commented code

File: utilities/aws-upload.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
+'use strict';
+
+var k = require('../config/constants.js');
+var AWS = require('aws-sdk');
+
+// var bucketName = 'my_bucket';
```
Comment (abhilashsajeev) [inline utilities/aws-upload.js]: remove

**Comment by ghost** [comment]
@abhilashsajeev Fixed

File: routes/index.js [hunk 6]
```diff
@@ -2,12 +2,49 @@
...
+        var filePath = 'JSON_Files/' + obj[category].trim().replace(/\s>\s/g, '/');
+        var jsonFilePath = filePath + '/' + obj[categoryId] + '.json';
+        delete obj[categoryId];
+        delete obj[category];
+        awsUpload(obj, jsonFilePath);
+      }
+
+    });
   });
+  //console.log(count);
```
Comment (abhilashsajeev) [inline routes/index.js]: remove

**Comment by abhilashsajeev** [comment]
@nithinp There are still commented codes.

**Pull request merged**